pallets/simple_democracy/src/lib.rs

```diff
@@ -0,0 +1,334 @@
+//! This is a facade over Substrate's democracy pallet offering similar functionality with a few differences.
+//! This pallet implements a "simple majority" governance where only Council can vote but anyone can
+//! propose by locking a fixed amount of tokens, `PublicProposalDeposit`. Similar to Substrate's governance pallet, both Council and
+//! general public take turns proposing, and a proposal follows "propose" -> "referendum" -> "enact" sequence.
```

Comment:
> both Council and general public take turns proposing

How/where are turns implemented?

Reply:
https://github.com/docknetwork/dock-substrate/pull/91/files#diff-5dabdfd4e3110162e036bcf65bc1a7c7bfaf86a1a348360d30b0585c1c5b3a7bR1593

Reply:
That link opens up an entire diff. Can you be more specific?

Reply:
GH is not expanding the file, i was pointing to `launch_next`